apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx

```diff
@@ -84,8 +74,10 @@ export default function FloatingScrollbar({
   }, [childContainerRef, scrollVisible, setScrollVisible]);
 
   React.useEffect(() => {
-    window.addEventListener('scroll', e => handleScrollAndResize(e));
-    window.addEventListener('resize', e => handleScrollAndResize(e));
+    window.addEventListener('scroll', () => handleScrollAndResize());
```

Comment:
Do you need to wrap this function for some reason or would this work just as well?
```suggestion
    window.addEventListener('scroll', handleScrollAndResize);
```